src/transient-error-lookup.js

```diff
@@ -0,0 +1,17 @@
+class TransientErrorLookup {
```

Comment:
This makes it easier and cleaner to stub it out in the tests. Add a comment to clarify that.